src/pages/TopicSelect.jsx

```diff
@@ -0,0 +1,23 @@
+//import ArticleList from "../components/article/ArticleList";
+import Header from "../components/Header";
+import Nav from "../components/Nav";
+import Topic from "../components/topic/Topic";
+import User from "../components/user/User";
+
+function TopicSelect({ topic, setTopic, topics, setTopics }) {
+  return (
+    <div>
+      <Header />
+      <Nav />
+      <User />
```

Comment:
If all pages/views have the Header, Nav and User components, you should place these outside of your Router in App (ie. before line 13 in App). This way your code will be drier, and they won't refresh on each different page load.